variables.tf

```diff
@@ -255,6 +255,18 @@ variable "check_s3_bucket_ssl_requests_only" {
   default     = true
 }
 
+variable "check_mfa_enabled_for_iam_console_access" {
```

Comment:
This one is problematic as a default as it's common to have robot IAM users that can't be configured with MFA. I'm okay with adding the rule but we should set the default to `false`.

Reply:
Okay, but this rule is only for users with "console access", so shouldn't the robot users not have console access?

Reply:
@dynamike I disabled both the rules by default and sync'd with master.  Do you think we could get this merged?